src/coreclr/pal/src/file/file.cpp

```diff
@@ -3434,6 +3434,8 @@ GetTempFileNameW(
     }
 
 done:
+    free(tempfile_name);
```

Comment:
Can the existing free above be removed then?
https://github.com/dotnet/runtime/pull/47477/files#diff-5084330bf89826abcc8087f997358a52e83581ccf861f31b66ac4e2dbf99211dR3416-R3417